integration-tests/integration/createIncident/view-your-report.cy.js

```diff
@@ -16,8 +16,46 @@ context('A reporter views their own report', () => {
     cy.task('stubUserDetailsRetrieval', ['MR_ZAGATO', 'MRS_JONES', 'TEST_USER'])
   })
 
+  it("A user's report should contain the incident location description even if DB record only contains the nomis format (i.e integer format) of the location's Id", () => {
```

Comment:
this is verifying that even if the report only contains the prisonApi location id, as is the case for existing reports, the locations description will still be displayed at the /view-report endpoint